src/test/java/core/basesyntax/service/impl/ValidatorImplTest.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.Validator;
+import org.junit.Test;
+
+public class ValidatorImplTest {
+    private final Validator validator = new ValidatorImpl();
+    
+    @Test
+    public void validateLine_validLine_ok() {
+        boolean expected = true;
+        String line = "p,apple,200";
+        boolean actual = validator.validateLine(line);
+        assertEquals(expected, actual);
+    }
+    
+    @Test
+    public void validateLine_invalidLine_notOk() {
+        boolean expected = false;
+        String line = "qwe,rtyu,uuuuu";
+        boolean actual = validator.validateLine(line);
+        assertEquals(expected, actual);
+    }
+}

```

Comment:
I think we need to test next situation:
"p,banana"
",banana,20"
"p,banana,p2p"
"p,,20"
"p,banana,"
"p,banana,20,50"
"P,banana,20"